amplify.yml

```diff
@@ -4,6 +4,7 @@ applications:
       phases:
         preBuild:
           commands:
+            - rm -rf node_modules
```

Comment:
Is this still necessary?

Reply:
It's not in the future, but we need to clean the cache at this moment. I'll make a follow-up PR to remove this line